helm_deploy/hmpps-interventions-service/Chart.yaml

```diff
@@ -6,5 +6,5 @@ version: 0.1.0
 
 dependencies:
   - name: generic-data-analytics-extractor
-    version: 0.1.3
+    version: 0.1.4
```

Comment:
This change brings in https://github.com/ministryofjustice/hmpps-helm-charts/pull/85
```diff
- args: ["extract_psql_all_tables_to_jsonl.sh && transfer_local_to_s3.sh"]
+ args: ["extract_table_names.py && extract_pg_jsonl_snapshot.py && transfer_local_to_s3.sh"]
```